### PR TITLE
Dev2

### PR DESCRIPTION
**Pull request by roytman**
## Why are these changes needed?
Several more doc changes

### CODE REVIEW

**Review by blublinsky — APPROVED** [review]

File: kind/README.md [hunk 1]
```diff
@@ -62,8 +62,7 @@ kubectl apply -f $ROOT_DIR/hack/kfp_ingress.yaml
 kubectl apply -f $ROOT_DIR/hack/minio_ingress.yaml
 ```
 
-Open the Kubeflow Pipelines UI at  http://localhost:8080/kfp/
-Note: pay attention to the slash at the end of the URL.
+Open the Kubeflow Pipelines UI at  http://localhost:8080
```
Comment (blublinsky) [inline kind/README.md]: Wait, where is the ingress to do this?

**Comment by revit13** [comment]
Please also fix the `Two important development tools will also be installed using the steps below: -[pre-commit](https://pre-commit.com/) -[twine](https://twine.readthedocs.io/en/stable/)` - it is not shown well. Thanks

File: README.md [hunk 2]
```diff
@@ -151,24 +151,15 @@ that can run on your machine. This implementation can also be extended to connec
...
-make setup
-```
-from this main package directory or from the `kind` directory.
-
-When you finish working with the cluster, you can destroy it by running,
-```bash
-make clean
-```
+When you finish working with the cluster, and want to clean up or destroy it. See the 
+[clean up the cluster](../kfp/doc/setup.md#cleanup)
```
Comment (revit13) [inline README.md:163]: should it be `#cleanupthecluster`, also in kfp/doc/setup.md (the link did not work for me)

Comment (roytman) [reply README.md:163]: no, it is correct.

**Review by revit13 — APPROVED** [review]

File: kind/README.md (hunk 1 above)
Comment (roytman) [reply kind/README.md]: yes, I returned it back

**Comment by roytman** [comment]
> Please also fix the `Two important development tools will also be installed using the steps below: -[pre-commit](https://pre-commit.com/) -[twine](https://twine.readthedocs.io/en/stable/)` - it is not shown well. Thanks

fixed, thanks

**Pull request merged**